frontends/p4/typeChecking/bindVariables.cpp

```diff
@@ -12,8 +12,7 @@ const IR::Type* BindTypeVariables::getP4Type(const IR::Type* type) const {
     if (type == nullptr)
         return type;
     auto rtype = type->getP4Type();
-    CHECK_NULL(rtype);
-    return rtype;
+    return rtype;  // This may be nullptr
```

Comment:
Yes, there is exactly one call of this method.
But as you can see from two lines above, this method could already return nullptr.